spec/models/basket_decorator_spec.rb

```diff
@@ -13,6 +13,17 @@
     end
   end
 
+  describe '#item_count' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.